src/patternfly/demos/LoginPage/examples/login-page-simple-example.hbs

```diff
@@ -12,20 +12,97 @@
+                    {{#> check}}
+                      {{#> check-input check-input--attribute='type="checkbox" id="login-demo-checkbox" name="login-demo-checkbox"'}}{{/check-input}}
+                      {{#> check-label check-label--attribute='for="login-demo-checkbox"'}}Keep me signed in for 30 days.{{/check-label}}
+                    {{/check}}
+                  {{/toolbar-item}}
+                {{/toolbar-group}}
+              {{/toolbar}}
+            {{/form-group}}
+          {{/form}}
         {{/login-box-body}}
         {{#> login-box-footer}}
-          Login box footer. Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure labore ipsa nemo nostrum id culpa atque nulla expedita at! Explicabo aut minima alias. Officia magnam est at omnis iste eius?
+          <p class="pf-u-mb-md">Or log in with</p>
+          {{#> list list--modifier="pf-m-grid"}}
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <img src="/assets/images/logo-google.svg" alt="Google logo" />
+                <p class="pf-u-ml-sm">Google</p>
+              {{/display}}
+            </li>
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <img src="/assets/images/logo-github.svg" alt="GitHub logo" />
+                <p class="pf-u-ml-sm">GitHub</p>
+              {{/display}}
+            </li>
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <img src="/assets/images/logo-dropbox.svg" alt="Dropbox logo" />
+                <p class="pf-u-ml-sm">Dropbox</p>
+              {{/display}}
+            </li>
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <img src="/assets/images/logo-facebook.svg" alt="Facebook logo" />
+                <p class="pf-u-ml-sm">Facebook</p>
+              {{/display}}
+            </li>
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <img class="" src="/assets/images/logo-gitlab.svg" alt="GitLab logo" />
+                <p class="pf-u-ml-sm">GitLab</p>
+              {{/display}}
+            </li>
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <p class=""><a href="https://www.patternfly.org/">More...</a></p>
+              {{/display}}
+            </li>
+          {{/list}}
         {{/login-box-footer}}
       {{/login-box}}
     {{/login-main}}
     {{#> login-footer}}
-        <a href="#">Login Footer</a>
+      {{#> list list--modifier="pf-m-inline"}}
+        <li>Terms of Use</li>
```

Comment:
I'm guessing these should be links, too. 

And this issue might be outside the scope of this PR, but if you were to add `<a>` elements here, I'm also guessing that the blue on the image background would not pass contrast requirements. 

@andresgalante Is the expectation that the Login layout handle styles for these links?

Reply:
You're right, fixed, thank you! The contrast probably is too low. @andresgalante should we add a variable for the color of footer links in the LoginBox component's CSS?